configs/model/divaepp.yaml

```diff
@@ -0,0 +1,11 @@
+# @package _group_
+model_type: DiVAEPP
+activation_fct: tanh
+beta_smoothing_fct: 4
+decoder_hidden_nodes: 
+  - 200
+  - 200
+n_encoder_layer_nodes: 200
+n_encoder_layers: 2
+n_latent_hierarchy_lvls: 4
+n_latent_nodes: 100
```

Comment:
This is the configuration file for your model. Simply specify `divaepp` in the top-level config `configs/config.yaml` and the modelCreator (see below) will load the DiVAEPP model and set the parameters from here.